Documentation/Readme.md

```diff
@@ -0,0 +1,3 @@
+# Documentation

```

Comment:
Is this file necessary if no content is included? did you create it so github shows some info when navigating to the subfolder?

Reply:
Basically some documentation is expected in the folder documentation. But there are only images used in other readme/documentation. So I added this note. Maybe it's a good idea to rename the folder?